Lib/idlelib/configdialog.py

```diff
@@ -1853,6 +1853,219 @@ def save_all_changed_extensions(self):
             self.ext_userCfg.Save()
 
 
+class FontPage:
+
+    def __init__(self, parent, tab_pages, highlight_sample):
```

Comment:
One other thing that I wasn't sure about - whether to send all of tab_pages to FontPage or to just send the frame as the parameter.  The main reason is this line in FontPage:

 frame = self.tab_pages.pages['Fonts/Tabs'].frame

Since the `pages['Fonts/Tabs']` is defined in ConfigDialog.create_widgets invokes TabbedPageSet, I'm not sure if it's correct to reference it within another class. 

So, then the init would be `def __init__(self, parent, frame, highlight_sample):`

and the instantiation would be `font_page = FontPage(self.parent, self.tab_pages.pages['Fonts/Tabs'].frame, self.highlight_sample)`